1.10.10/alpine3.10/Dockerfile

```diff
@@ -21,12 +21,13 @@ ARG VERSION="1.10.10-6.*"
 ARG SUBMODULES="all, statsd, elasticsearch"
 ARG AIRFLOW_MODULE="astronomer_certified[${SUBMODULES}]==$VERSION"
 ARG REPO_BRANCH=master
+ENV AIRFLOW_VERSION="1.10.10"
```

Comment:
Why make this an env?

Reply:
Updated in updated in https://github.com/astronomer/ap-airflow/pull/160/commits/28339144b747aadea11ed8b215697b8acc4b805d